Adding Nano Bots Package

icebaker opened the pull request:
<!--
The manual review may take several days or weeks,
depending on the reviewer's availability and workload.
Patience padawan!

You can request a review from @packagecontrol-bot.
Please ensure the reviews pass and follow any instructions.

Please provide some information via this checklist,
feel free to remove what't not applicable.
-->

- [x] I'm the package's author and/or maintainer.
- [x] I have have read [the docs][1].
- [x] I have tagged a release with a [semver][2] version number.
- [x] My package repo has a description and a README describing what it's for and how to use it.
- [x] My package doesn't add context menu entries. *
- [x] My package doesn't add key bindings. **
- [x] Any commands are available via the command palette.

My package connects Sublime Text to Artificial Intelligent Bots created following the [_Nano Bots_](https://icebaker.github.io/nano-bots/#/README) specification, which supports different AI Large Language Models providers. This enables direct use of the user's custom AI bots inside the editor.

More details at:
- https://github.com/icebaker/sublime-nano-bots
- https://icebaker.github.io/nano-bots-spec/#/README

There are no packages like it in Package Control.


<!-- 
*)   Unless it definitely really needs them,
     they apply to the cursor's context
     and their visibility is conditional.
     Space in this menu is limited!
**)  There aren't enough keys for all packages,
     you'd risk overriding those of other packages.
     You can put commented out suggestions in a keymap file, 
     and/or explain how to create bindings in your README.
***) We have hundreds of color schemes,
     and plenty of scopes to make any syntax work. 

For bonus points also considered how the review guidelines apply to your package:
https://github.com/wbond/package_control_channel/wiki#reviewing-a-package-addition

For updates to existing packages:
If your package isn't using tag based releases,
please switch to tags now.
 -->

[1]: https://packagecontrol.io/docs/submitting_a_package
[2]: https://semver.org
[3]: https://www.git-scm.com/docs/gitattributes#_export_ignore

icebaker commented:
@packagecontrol-bot

icebaker commented:
@packagecontrol-bot review

icebaker commented:
@packagecontrol-bot review

braver commented:
Thanks for your submission. Looks well considered and clearly a lot of effort has been put into for instance the readme. I have only a few topics I'd like to touch on here.

For one, we've been asking all packages that may (in some cases, in some way, etc) upload code to third party services. It might not always be clear to all users if or when that happens, and they might be working on code or text where that's a problem. I'm not sure how that applies here, but just take that as our blanket advice for anything that "smells like AI" 😅 

Why are you using the `.no-sublime-package` file? Nothing in your code jumps out to me as something that requires it. I was also surprised to see content in that file. If you want to exclude files from the final package as it's delivered to users, you can use [.gitattributes](https://www.git-scm.com/docs/gitattributes) (the packaging system relies on git for a lot of the work).

braver commented on repository/n.json:
```diff
@@ -34,6 +34,18 @@
+		{
+			"name": "Nano Bots",
+			"details": "https://github.com/icebaker/sublime-nano-bots",
+			"issues": "https://github.com/icebaker/sublime-nano-bots/issues",
```
Since this is the default location for issues you can leave it out, package control will figure that out for you.

icebaker commented:
Hey @braver, thanks for your feedback!

> "Smells like AI"

Indeed, it is definitely AI!

Based on that, I've added a [Privacy and Security: Frequently Asked Questions](https://github.com/icebaker/sublime-nano-bots#privacy-and-security-frequently-asked-questions) section to the README. I've also highlighted it at the start of the [README](https://github.com/icebaker/sublime-nano-bots#readme) to ensure transparency and make users aware of it. The goal is to provide clear information about privacy aspects when using this. Do you think this is a good way to tackle this?

> - [Privacy and Security: Frequently Asked Questions](https://github.com/icebaker/sublime-nano-bots#privacy-and-security-frequently-asked-questions)
>    - [Will my files/code/content be shared or uploaded to third-party services?](https://github.com/icebaker/sublime-nano-bots#will-my-filescodecontent-be-shared-or-uploaded-to-third-party-services)
>    - [What information may be shared with third-party AI providers?](https://github.com/icebaker/sublime-nano-bots#what-information-may-be-shared-with-third-party-ai-providers)
>    - [Who are these third parties?](https://github.com/icebaker/sublime-nano-bots#who-are-these-third-parties)
>    - [Is there an option to avoid sharing any information?](https://github.com/icebaker/sublime-nano-bots#is-there-an-option-to-avoid-sharing-any-information)
>    - [Can I use this for private or confidential content/code?](https://github.com/icebaker/sublime-nano-bots#can-i-use-this-for-private-or-confidential-contentcode)
>    - [Do I need to pay to use this?](https://github.com/icebaker/sublime-nano-bots#do-i-need-to-pay-to-use-this)
>    - [Is this project affiliated with OpenAI?](https://github.com/icebaker/sublime-nano-bots#is-this-project-affiliated-with-openai)
>   - [Warranty and Disclaimer](https://github.com/icebaker/sublime-nano-bots#warranty-and-disclaimer)

> .no-sublime-package

Yeah, I don't really need this file or a .gitattributes one since there's no file I need to exclude from the package. It's gone.

> Default location for issues

Gotcha, removed the unnecessary key.

braver commented:
Package control, and the review bot, won’t see your changes until you tag a release.

icebaker commented:
@braver Done, the tag has been created and released.
